src/api/github_labels.js

```diff
@@ -0,0 +1,6 @@
+import axios from 'axios';
+//this is only temporary need to refactor into 1 component but function is working
+export default  axios.create({
+    baseURL: 'https://api.github.com/repos/vuejs/vue/labels',
+   
+  })
```

Comment:
Create a axios base so you dont create axios instance again and again. Follow DRY process.

Your .env
```
REACT_APP_GITHUB_API=https://api.github.com/repos/vuejs/vue
```

```suggestion
export default  axios.create({
  baseURL: `${process.env.REACT_APP_GITHUB_API}`,
})
```

then when fetching, you can reuse your axios base like this.
```
await API.get('/labels');
await API.get('/issues');
```